at.nis.one_click_backup.yml

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "at.nis.one_click_backup",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "command": "NIS_One-Click-Backup_Qt",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=host",
```

Comment:
WHy do you need host? This is a very broad access

Reply:
Because this program is (primarily) meant for backing up your folders to an external drive. Giving only `home`-permission would break this functionality, because it would deny the program access to the mount-folders for external drives (e. g. `/media/<user>/` )